test/main.test.js

```diff
@@ -2,7 +2,7 @@
 import { rejects, strictEqual, deepStrictEqual, match } from "assert";
 import { test, before, describe } from "mocha";
 // eslint-disable-next-line no-unused-vars
-import _init, { initSync, __wasm, init } from "../main.js";
+import _init, { init } from "../module.js";
```

Comment:
I think the is the appropriate change, but the tests fail with `Unknown file extension ".wasm" for /Users/aaron/Code/textile/wasm-sqlparser/main.wasm`.